src/database/utilities/createRecord.js

```diff
@@ -17,15 +17,20 @@ import { SETTINGS_KEYS } from '../../settings';
  * already exists).
  *
  * @param   {Realm}         database  The local database.
+ * @param   {string}        id        Id of the address (can be undefined).
  * @param   {string}        line1     Line 1 of the address (can be undefined).
  * @param   {string}        line2     Line 2 of the address (can be undefined).
  * @param   {string}        line3     Line 3 of the address (can be undefined).
  * @param   {string}        line4     Line 4 of the address (can be undefined).
  * @param   {string}        zipCode   Zip code of the address (can be undefined).
  * @return  {Realm.object}            The Address object described by the params.
  */
-export const getOrCreateAddress = (database, line1, line2, line3, line4, zipCode) => {
+export const getOrCreateAddress = (database, id, line1, line2, line3, line4, zipCode) => {
```

Comment:
I believe this is going to break the calls in `incomingSyncUtils.js`? Might want to make it the last param? possibly having an object param, but quite a few changes over just making it the last param

Reply:
Ah, forgot to make the changes to the other callers, plan was to just add a `null` param, but that's pretty ugly. Tbh, I think longer term this method can be made a bit nicer, but I think for now making an object param is nicer, it's not called in that many places!

Reply:
cfafe1f96cbfb8f91bc811b521b2b9c81542954b.